### PR TITLE
Added CPU as user-defined variable

### LINKED ISSUE #67: Add CPU to resources
## Feature description
Add CPU as a variable you can set in the module

## Why is it needed?
More flexibility and control for the user

## Suggestion(s)/solution(s) [Optional]
add to `variables.tf`

## Definition of done
User can set CPU from input variable to module

## Checklist (after created issue)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### PR DESCRIPTION
**Pull request by pdmthorsrud**
## Closes/fixes/resolves issue(s)?
closes #67 

## What was added/changed/fixed?
CPU as user-defined variable

## Checklist (after created PR)
- [x] Added reviewers
- [x] Added assignee(s)
- [x] Added label(s)
- [x] Added to project
- [x] Added to milestone

### CODE REVIEW

File: conf/nomad/presto.hcl [hunk 1]
```diff
@@ -385,6 +385,7 @@ EOF
       }
       resources {
         memory = ${memory}
+        cpu = ${cpu}
```
Comment (claesgill) [inline conf/nomad/presto.hcl:388]:
```suggestion
        memory = ${memory}
        cpu    = ${cpu}
```
# :shrug:

**Review by claesgill — APPROVED** [review]
LGTM!

File: conf/nomad/presto.hcl (hunk 1 above)
Comment (zhenik) [reply conf/nomad/presto.hcl:388]: I wonder does `make pre-commit` help in that context. [link to command](https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/Makefile#L100)?

**Review by zhenik — COMMENTED** [review]
Could you update `input` and `example` sections in documentation?

File: conf/nomad/presto.hcl (hunk 1 above)
Comment (pdmthorsrud) [reply conf/nomad/presto.hcl:388]: Huh, good point. I'll test it out now!

**Comment by pdmthorsrud** [comment]
> Could you update `input` and `example` sections in documentation?

Crap, yep, on it. Documentation will be the death of me one day.

File: conf/nomad/presto.hcl (hunk 1 above)
Comment (claesgill) [reply conf/nomad/presto.hcl:388]: > I wonder does `make pre-commit` help in that context. [link to command](https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/Makefile#L100)?

Yes, that should do the trick!

**Comment by zhenik** [comment]
> > Could you update `input` and `example` sections in documentation?
> 
> Crap, yep, on it. Documentation will be the death of me one day.

We have changelog enforcer in CI, maybe we can do smth similar with root README.md?

**Comment by pdmthorsrud** [comment]
> > > Could you update `input` and `example` sections in documentation?
> > 
> > 
> > Crap, yep, on it. Documentation will be the death of me one day.
> 
> We have changelog enforcer in CI, maybe we can do smth similar with root README.md?

Would it not be fair to say that there is _at least_ a 50/50 whether the README needs changing or not? Maybe even a 70/30

**Comment by claesgill** [comment]
> > > > Could you update `input` and `example` sections in documentation?
> > > 
> > > 
> > > Crap, yep, on it. Documentation will be the death of me one day.
> > 
> > 
> > We have changelog enforcer in CI, maybe we can do smth similar with root README.md?
> 
> Would it not be fair to say that there is _at least_ a 50/50 whether the README needs changing or not? Maybe even a 70/30

Agree with you @pdmthorsrud. We could add it to the checklist in the [pull_request_template.md](https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/.github/pull_request_template.md)  though. Or WDYT?

**Comment by pdmthorsrud** [comment]
> Agree with you @pdmthorsrud. We could add it to the checklist in the [pull_request_template.md](https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/.github/pull_request_template.md) though. Or WDYT?

I'm a little iffy on adding more checkmarks to the template. I don't like crossing off everything :D That is a personal preference though, and I'll be honest, it will probably help me remember

**Comment by claesgill** [comment]
> > Agree with you @pdmthorsrud. We could add it to the checklist in the [pull_request_template.md](https://github.com/fredrikhgrelland/terraform-nomad-presto/blob/master/.github/pull_request_template.md) though. Or WDYT?
> 
> I'm a little iffy on adding more checkmarks to the template. I don't like crossing off everything :D That is a personal preference though, and I'll be honest, it will probably help me remember

I agree, but it would be a good reminder that, in the long run, saves us time 🤑 

It doesn't need to be a checkbox. Could be smtl:
- Need to update README(s)?

**Comment by pdmthorsrud** [comment]
> I agree, but it would be a good reminder that, in the long run, saves us time 🤑
> 
> It doesn't need to be a checkbox. Could be smtl:
> 
> * Need to update README(s)?

Okok, I'm down! It's a good idea

**Review by zhenik — APPROVED** [review]

**Review by claesgill — APPROVED** [review]

**Review by dangernil — APPROVED** [review]

**Review by zhenik — APPROVED** [review]
LGTM

**Comment by pdmthorsrud** [comment]
@dangernil or @claesgill when you have time, please approve :+1:

**Review by claesgill — APPROVED** [review]

**Pull request merged**